### ISSUE TITLE
tasks_endpoint attempts to use pagination

### ISSUE THREAD

**Issue by cjculver** [tableau/server-client-python#294]
First time filing an issue on GitHub, so I apologize if it's formatted incorrectly (or just wrong)...

I am attempting to create functions that will get the associated resources. In most cases, I would use something along the lines of:

```
def get_schedules():
    for schedule in TSC.Pager(server.schedules):
        print("\nSchedule Name: {}\nSchedule ID: {}".format(schedule.name, schedule.id)
```

However, the 'Get Extract Refresh Tasks API call does not support pagination. As a result, I've created my function like so:

```
def get_refresh_tasks():
    for task in server.tasks.get():
        task_list = "\nTask ID: {}\nSchedule ID: {}\nRefresh Priority: {}\nExtract type: {}\nExtract ID: {}"
        print(task_list.format(task.id, task.schedule_id, task.priority, task.target.type, task.target.id))
```
This results in a AttributeError: 'list' object has no attribute 'id'. But I found that by commenting out two parts of the modified the tasks_endpoint.py file, my code worked entirely as expected:
```
    def get(self, req_options=None):
        logger.info('Querying all tasks for the site')
        url = self.baseurl
        server_response = self.get_request(url, req_options)
        #pagination_item = PaginationItem.from_response(server_response.content, self.parent_srv.namespace)
        all_extract_tasks = TaskItem.from_response(server_response.content, self.parent_srv.namespace)
        return all_extract_tasks#, pagination_item

```

As an alternative workaround, the following function works just fine AS-IS:

```
def get_refresh_tasks():
    tasks = server.tasks.get()
    for task in tasks[0]:
        task_list = "\nTask ID: {}\nSchedule ID: {}\nRefresh Priority: {}\nExtract type: {}\nExtract ID: {}"
        print(task_list.format(task.id, task.schedule_id, task.priority, task.target.type, task.target.id))

```
Am I correct in assuming the tasks_endpoint file should not return pagination_item since the Get Extracts Tasks API call does not support pagination? Or should I be writing the the get_refresh_tasks function like the last code example

**Comment by cjculver** [comment]
I suppose it could also be fixed with a server-side enhancement by including pagination on the REST API's [Get Extract Refresh Tasks](https://onlinehelp.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref.htm#Get_Extract_Refresh_Tasks) method, and commenting out the lines on the tasks_endpoint would simply be a workaround in the meantime.

**Comment by t8y8** [comment]
@RussTheAerialist or @shinchris any thoughts on this? I would expect tasks to be paginated on the Server side

**Comment by irwando** [comment]
Please try this again.  The fix proposed in #320 was long ago released.  Let us know if you still have issues.
